src/Modules/StatCan.OrchardCore.Assessment/Views/Widget-Assessment.liquid

```diff
@@ -0,0 +1,46 @@
+<div id="surveyElement" style="display:inline-block;width:100%;"></div>
```

Comment:
You should not use an id in a widget, a classname is safer.

Reply:
I don't think surveyjs's render function can anything but an `id`.